src/simplewallet/simplewallet.cpp

```diff
@@ -7786,6 +7889,49 @@ bool simple_wallet::import_key_images(const std::vector<std::string> &args)
   return true;
 }
 //----------------------------------------------------------------------------------------------------
+bool simple_wallet::hw_key_images_sync(const std::vector<std::string> &args)
+{
+  if (!m_wallet->key_on_device())
+  {
+    fail_msg_writer() << tr("command only supported by HW wallet");
+    return true;
+  }
+  if (!m_wallet->get_account().get_device().has_ki_cold_sync())
+  {
+    fail_msg_writer() << tr("hw wallet does not support cold KI sync");
+    return true;
+  }
+  if (!m_wallet->is_trusted_daemon())
+  {
+    fail_msg_writer() << tr("this command requires a trusted daemon. Enable with --trusted-daemon");
+    return true;
+  }
+
+  LOCK_IDLE_SCOPE();
+  try
+  {
+    message_writer(console_color_white, false) << "\r" <<
+                                               tr("Please confirm the key image sync on the device");
```

Comment:
Why the \r, out of curiosity ? This seems to be a user command, so there wouldn't be anything to overwrite AFAICT.

Reply:
Hmm true, I've removed it.